#!/usr/bin/python3
def raise_exception_msg(message=""):
    try:
        geek
        print(geek)
    except NameError:
        raise NameError(message)
